api_internal.go

```diff
@@ -367,9 +367,67 @@ func (c *Couchbase) uploadClusterCACert(pem []byte) error {
 	return c.n_post("/controller/uploadClusterCA", pem, nil, headers)
 }
 
+func (c *Couchbase) getClusterVersion() (v string, err error) {
+	info, err := c.getPools()
+	if err != nil {
+		return
+	}
+	arr := strings.Split(info.ImplVersion, "-")
```

Comment:
I'm not sure about getting "ImplVersion"
This may be specific to a node. 

For XDCR, we use "clusterCompatibility" key from "pools/default", which is per cluster.

Reply:
Addressed in https://github.com/couchbase/gocbmgr/pull/6/commits/0695e6519f44c7d1a327792343fec5408530cd43